content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -1,21 +1,24 @@
 
-### NMDC Field notes
-The NMDC Field Notes app is designed to bring standard metadata capture into the field. This app is built following the same schema format as the Submission portal and also supports community standards. When using the NMDC Field Notes app, metadata can be captured for your samples while in the field using in-app tools and formatting. The NMDC Field Notes app can be used offline and provides a direct connection to the NMDC Submission Portal so your metadata is already there upon field work completion.
+### NMDC Field Notes mobile app
+The NMDC Field Notes mobile app is designed to support real-time registration and collection of standardized metadata and field measurements. This mobile app is built following the same schema format as the Submission Portal and similarly supports the community standards [described above](### NMDC Submission Portal) as well as in [Collaboration to support Community Standards](### Collaboration to support Community Standards) below. When using the NMDC Field Notes mobile app, metadata can be captured for your samples while in the field using in-app tools and formatting. The NMDC Field Notes mobile app can be used offline and provides a direct connection to the NMDC Submission Portal so your metadata is already there upon field work completion.
+
+#### Connecting NMDC Field Notes and the NMDC Submission Portal
+The NMDC Field Notes mobile app synchronizes data with the NMDC Submission Portal and vice versa. This means that changes made in one place will be reflected in the other. Additional details of this synchronization and important considerations are captured in the Field Notes mobile app [documentation and guide](https://github.com/microbiomedata/nmdc-field-notes/blob/main/src/pages/GuidePage/md/guide-page.md).
```

Comment:
Ah I didn't realize you'd be linking to this Markdown file. I considered it to be more of a source code file than a documentation file. I'm not saying you have to change anything. This is mostly a note to myself that we need to be aware of this link if we ever want to move this file around in the source tree.